app/views/quizzes/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Editing quiz</h1>
```

Comment:
we don't need this right? if so, can you remove this file?

Reply:
Will remove